sdk/appconfiguration/tests.yml

```diff
@@ -14,4 +14,8 @@ jobs:
       ServiceDirectory: appconfiguration
       EnvVars:
         APP_CONFIG_CONNECTION: $(python-appconfiguration-connection-string)
+        APP_CONFIG_ENDPOINT: $(python-appconfiguration-endpoint-string)
+        AZURE_CLIENT_ID: $(python-appconfiguration-client-id-string)
```

Comment:
Live tests should be using `aad-azure-sdk-test-*` instead of creating many separate apps... That app should be given the same permissions to the endpoint at `$(python-appconfiguration-endpoint-string)` that the current `$(python-appconfiguration-client-id-string)` has.